### PR TITLE
Add blog post for FOSDEM talk by Orlix

### PR DESCRIPTION
**Pull request by OrlinVasilev**
Signed-off-by: OrlinVasilev <ovasilev@vmware.com>

### CODE REVIEW

File: content/blog/harbor-at-fosdem-2022.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  * Virtual Maintainers track 
+  * Virtual Office Hours
+  * In-person Kiosk 
+  * Breakfast anyone?
+
+### KubeCon NA 2020 - Detroit
+  * [Call for Proposals](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/cfp/)
+  * Keep you posted how we can meet there!
+  * If you want to join us on all events or you need help submitting your proplsal please do not hesitate and [contact me on Slack](https://cloud-native.slack.com/archives/D7EF6P9C5).
+
```
Comment (a-mccarthy) [inline content/blog/harbor-at-fosdem-2022.md:43]: Are there any events in China that we are planning to be at that we should include? or meet-ups or anything?

File: content/blog/harbor-at-fosdem-2022.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+description: "Harbor - The Container Registry"
+date: 2022-03-24T12:00:00+04:00
+showPageInfo: true
+---
+
+## Harbor - The Container Registry at FOSDEM 2022!
+
+I'm excited to share that I recently had the opportunity to represent the Harbor community at [FOSDEM 2022](https://fosdem.org/2022/). As a longtime FOSDEM attendee (my 10th-ish year in a row), this was my first time speaking at the conference. It was a dream come time to be accepted to the [Containers devroom track](https://fosdem.org/2022/schedule/track/containers/) and talk with my fellow attendees about the Harbor project.
+​
+As everything during the pandemic this year's FOSDEM was 100% virtual. Next year I hope we can meet each other in person at the venue, but until then below is the link to the recorded talk and slides.
```
Comment (a-mccarthy) [inline content/blog/harbor-at-fosdem-2022.md:14]: this needs a space or new line between `Harbor project.As ` its showing up without a space in the preview.

File: content/blog/harbor-at-fosdem-2022.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+​
+Myself and the Harbor maintainers hope to see you there! Where possible, we'll attend in person, but may need to meet up with you all virtually during the event.
+​
+And as always, you are able to [join the Harbor community](https://goharbor.io/community/) at any time meet during our community meetings or in the [#harbor](https://cloud-native.slack.com/archives/CC1E09J6S) channel on the CNCF slack.
+​
+
+Hope to see you soon!  
+
+[Orlix](https://github.com/OrlinVasilev)
```
Comment (a-mccarthy) [inline content/blog/harbor-at-fosdem-2022.md:52]: Maybe add in your title as community manager?

File: content/blog/harbor-at-fosdem-2022.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+​
+It’s company and organization sponsored but represents only projects and communities not companies, which makes it unique in the Open Source world these days! As such it’s always packed with folks from around the globe, some representing their projects others just for the beer and waffles.
+
+![We are full](https://upload.wikimedia.org/wikipedia/commons/0/00/FOSDEM_2015_full_sign_1.jpg)
+​
+## Connect with the Harbor team in 2022
+​
+Harbor is a [healthy](https://all.devstats.cncf.io/d/53/projects-health-table?orgId=1) and vibrant Graduated project at CNCF. We’ve seen growth in our community and superb contributions in the form of issues, code, and project feedback. If you'd like to connect with us, the Harbor team currently plans to attend the following conferences and events this year.
+​
+### KubeCon EU 2022 - Valencia
```
Comment (a-mccarthy) [inline content/blog/harbor-at-fosdem-2022.md:33]: Maybe add links to these conference schedules? or where folks can get more information?

**Review by a-mccarthy — COMMENTED** [review]
just a few comments

File: content/blog/harbor-at-fosdem-2022.md (hunk 2 above)
Comment (OrlinVasilev) [reply content/blog/harbor-at-fosdem-2022.md:14]: Strange, the NL is there :) will add another one

File: content/blog/harbor-at-fosdem-2022.md (hunk 4 above)
Comment (OrlinVasilev) [reply content/blog/harbor-at-fosdem-2022.md:33]: good idea done!

File: content/blog/harbor-at-fosdem-2022.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  * Virtual Office Hours
+  * In-person Kiosk 
+  * Breakfast anyone?
+
+Full schedule for conference [here](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/program/schedule/)
+
+### KubeCon NA 2022 - Detroit - Oct 24 - 28
+  * [Call for Proposals](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/cfp/)
+  * Keep you posted how we can meet there!
+  * If you want to join us on all events or you need help submitting your proplsal please do not hesitate and [contact me on Slack](https://cloud-native.slack.com/archives/D7EF6P9C5).
```
Comment (a-mccarthy) [inline content/blog/harbor-at-fosdem-2022.md:47]: proplsal to proposal

File: content/blog/harbor-at-fosdem-2022.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+### KubeCon China 2022 - TBA
```
Comment (a-mccarthy) [inline content/blog/harbor-at-fosdem-2022.md:49]: Maybe add a line that says something like, "Event details are still being figured out". looking at the preview, it looks like the paragraph below is for this heading. https://deploy-preview-306--goharbor-io.netlify.app/blog/harbor-at-fosdem-2022/

File: content/blog/harbor-at-fosdem-2022.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+---
+title: "Harbor at FOSDEM 22"
+author:
+  name: "Orlin Vasilev"
+description: "Harbor - The Container Registry"
+date: 2022-03-24T12:00:00+04:00
```
Comment (a-mccarthy) [inline content/blog/harbor-at-fosdem-2022.md:6]: we should update the date

**Review by a-mccarthy — COMMENTED** [review]
found a few more small things.

**Review by a-mccarthy — APPROVED** [review]
lgtm!

**Pull request merged**